spec/config_map_spec.rb

```diff
@@ -15,10 +15,21 @@
 
   before do
     allow(source).to receive(:source_type).and_return(source_type)
-    allow(source).to receive(:[]) { |arg| arg == 'uid' ? 'source-1' : nil }
+    allow(source).to receive(:[]) do |arg|
+      case arg
+      when 'id' then '1'
+      when 'uid' then 'source-1'
+      when 'name' then 'Source 1'
+      end
```

Comment:
We should either do either
`let(:source) { SourcesApiClient::Source.new(:source_type => ..., :uid => ..., etc...)`
or
`let(:source) { OpenStruct.new(...) }`

I would argue for the first one, we already depend on SourcesApiClient as a dependency so stubbing everything doesn't reduce inter-dependency it would just cover up a change to SourcesApiClient breaking this repo.  If a dependency change breaks this repo I'd rather find out from a scheduled cron travis run then in production

Reply:
@agrare this source is not an `SourcesApiClient::Source`, but orchestrator's Source. But I get the point, I could simulate the Source's structure better there.

Reply:
@agrare changed to `Orchestrator::Source`. Btw we don't relay on SourcesApiClient in this repo, it's using RestClient instead.